elide-contrib/elide-dynamic-config-helpers/src/test/resources/elideSecuritySchema.json

```diff
@@ -44,9 +44,5 @@
             }
         }
     },
-    "required": [
```

Comment:
Why are we duplicating this file in the test resources?  Why not just use the one in the main resources?